source/initial-dns-seedlist-discovery/tests/README.md

```diff
@@ -3,6 +3,31 @@
 This directory contains platform-independent tests that drivers can use to prove their conformance to the Initial DNS
 Seedlist Discovery spec.
 
+### Prose Tests
+
+1. When given the following valid SRVs, parse the connection string without throwing a parse related error.
```

Comment:
```suggestion
1. When given the following valid SRVs, parse the connection string

2. assert the driver does not throw a parse related error.
```

I think the steps should be clear about what to check for, "without throwing" is a bit ambiguous, should the author squash that error and move on? etc.

Reply:
Yep! I'm thinking I'll change it to talk about the pre DNS validation step not causing any errors. Messaged you on slack to align with this.